package.json

```diff
@@ -13,10 +13,11 @@
   "dependencies": {
     "moment": "^2.17.1",
     "moment-timezone": "^0.5.11",
-    "react": "15.4.2",
-    "react-native": "^0.42.0",
+    "react": "16.0.0-alpha.6",
+    "react-native": "^0.44.0",
     "react-native-blur": "^2.0.0",
-    "react-native-code-push": "^1.17.0-beta",
+    "react-native-code-push": "^2.0.3-beta",
```

Comment:
This is the only upgrade I'm not sure about. Since the semver indicates a major breaking change, I'd prefer we leave `react-native-code-push` on `v1.17.0-beta`.